optimade/filtertransformers/mongo.py

```diff
@@ -92,6 +92,9 @@ def expression_phrase(self, arg):
             # without NOT
             return arg[0]
 
+        if list(arg[1].keys()) == ["$or"]:
+            return {"$nor": arg[1]["$or"]}
+
         # with NOT
         # TODO: This implementation probably fails in the case of `"(" expression ")"`
```

Comment:
Is this still true? I guess it is.

Reply:
Yeah, not sure. I don't think the example is valid for the grammar anymore?

Reply:
I have no idea either. Although the example _was_ put in for this grammar by @fekad I believe?